pkg/proto/configuration/server/server.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package buildbarn.configuration.server;
```

Comment:
I would suggest having the grpc and http/metrics listen ports in here too.

Reply:
Other components importing this then would import the ports message as well which could be a bit confusing right? Also right now configuration is consistent across components, so either this would break consistency or you would need a server.proto for each, but in some cases, this would be a bit overkill (e.g. [bb-scheduler configuration](https://github.com/buildbarn/bb-remote-execution/blob/master/pkg/proto/configuration/bb_scheduler/bb_scheduler.proto))